core/src/test/java/io/onetable/TestIcebergTable.java

```diff
@@ -240,8 +283,11 @@ public void reload() {
 
   @Override
   public List<String> getColumnsToSelect() {
+    // There is representation difference in hudi and iceberg for local timestamp micros field.
+    // and hence excluding it from the list of columns to select.
     return icebergDataHelper.getTableSchema().columns().stream()
         .map(Types.NestedField::name)
+        .filter(name -> !name.equals("timestamp_local_micros_nullable_field"))
```

Comment:
can we check if there is a way for Hudi to read this back as the logical type? Do the values read back as timestamps in Delta?

Reply:
For hudi there is a bug, linked in the comments and it doesn't work for Delta too and I see spark 3.2 has no support for that granularity and coverting to TimestampType doesn't work on the read side.